### ISSUE TITLE
Open editor in cleanlab-studio upon schema rejection

### ISSUE THREAD

**Issue by aayushchhabra1999** [cleanlab/cleanlab-studio#54]
When a user tries to upload a dataset, cleanlab studio proposes an auto-generated schema that the user may or may not accept. 

Currently: Upon rejection, the user can provide a schema with the `--schema` flag.

Enhancement: Upon rejection, the default editor (vim, nano, emacs, vscode) with a transient file should be opened in the user's system with the proposed schema and the user can edit this schema as needed.

**Comment by aayushchhabra1999** [comment]
@anishathalye Thinking that we should put a pause on this and the MVP PR that I opened for this till we finalize more details on the web-based schema flow. I think it is best to directly migrate to that web-based flow and put resources into that effort as compared to putting effort into complex editor flows and improvements in the CLI. What do you think?

**Comment by anishathalye** [comment]
Agreed, let’s put this on hold. On Nov 23, 2022, at 8:32 AM, Aayush Chhabra ***@***.***> wrote:﻿
@anishathalye Thinking that we should put a pause on this and the MVP PR that I opened for this till we finalize more details on the web-based schema flow. I think it is best to directly migrate to that web-based flow and put resources into that effort as compared to putting effort into complex editor flows and improvements in the CLI. What do you think?
